app/lib/shared/datastore_helper.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:gcloud/db.dart';
+import 'package:retry/retry.dart';
+
+class TransactionWrapper {
+  final Transaction _tx;
+  bool _mutated = false;
+
+  TransactionWrapper._(this._tx);
+
+  Future<List<T>> lookup<T extends Model>(List<Key> keys) =>
```

Comment:
let's add `@override` annotations here

Reply:
It's intentionally not a subclass..  I created the wrapper to hide `commit` and `rollback` since I don't think we need those.

Reply:
Ah, my bad.